fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -4656,7 +4656,7 @@ public void updateLoanOutstandingBalances() {
                 }
                 outstanding = outstanding.plus(transactionOutstanding);
                 loanTransaction.updateOutstandingLoanBalance(MathUtil.negativeToZero(outstanding.getAmount()));
-            } else {
+            } else if (!loanTransaction.isAccrualActivity()) {
```

Comment:
It would be better if add this to the `retrieveListOfTransactionsExcludeAccruals`

Reply:
agree, fixing